pkg/build/builder/util.go

```diff
@@ -26,3 +30,67 @@ func getBuildEnvVars(build *api.Build) map[string]string {
 	}
 	return envVars
 }
+
+type BuildConfigClientInterface interface {
+	GetBuildConfig(namespace, name string) (*buildapi.BuildConfig, error)
+	UpdateBuildConfig(buildConfig *buildapi.BuildConfig) error
+}
+
+type BuildClientInterface interface {
```

Comment:
Looking more into it, I haven't noticed this interface landed in `util.go`, sorry. I see a `BuildInterface` here which should contain create and update methods and the other two methods (`CreateBuildFromConfig`, `CreateBuildWithImmutableId`) are just plain util methods, that should not be part of that interface.